netmap/netmap_contract.go

```diff
@@ -85,6 +88,15 @@ func _deploy(data interface{}, isUpdate bool) {
 
 	if isUpdate {
 		common.CheckVersion(args.version)
+
+		data := storage.Get(ctx, "snapshotCurrent")
+		storage.Put(ctx, snapshotKeyPrefix+"0", data)
+
+		data = storage.Get(ctx, "snapshotPrevious")
+		storage.Put(ctx, snapshotKeyPrefix+"1", data)
```

Comment:
If I understand correctly, we put new epochs with incremented `id`. So previous snapshot should be with `id:0`, current snapshot with `id:1` and `snapshotCurrentIDKey:1`.

Reply:
Yes, fixed.